### ISSUE TITLE
feature request: detect hardcoded paths in AppRun

### ISSUE THREAD

**Issue by alex-tee** [TheAssassin/appimagelint#23]
my AppImage was failing because of a hardcoded rpath:

`/tmp/.mount_zrythmaLNdzc//usr/bin/zrythm: error while loading shared libraries: /usr/lib/zrythm/carla/libcarla_host-plugin.so: cannot open shared object file: No such file or directory`

related info from `readelf -d`:
```
0x000000000000001d (RUNPATH)            Library runpath: [/usr/lib/zrythm/carla]
0x0000000000000001 (NEEDED)             Shared library: [/usr/lib/zrythm/carla/libcarla_host-plugin.so]
```

I used `patchelf` to fix these issues. I think appimagelint should detect hardcoded paths like these

**Comment by alex-tee** [comment]
maybe related to #19
